TheaterSchedule.BLL/Services/MessageService.cs

```diff
@@ -30,30 +41,42 @@ public MessageDTO GetById(int id)
             {
                 return null;
             }
-            return new MessageDTO
-            {
-                MessageId = message.MessageId,
-                Subject = message.Subject
-            };
+
+            return _mapper.Map<MessageDTO>(message);
         }
 
-        public void SendMessage(MessageDTO newMessage)
+        public async Task SendMessage(MessageDTO newMessage)
         {
-            Account account = accountRepository.GetAccountByPhoneId(newMessage.PhoneId);
+            var account = userRepository.GetById(newMessage.AccountId);
+
             if (account == null)
             {
-                throw new ArgumentException("Non existent account");
+                throw new NullReferenceException();
```

Comment:
throwing manually of null reference exception is bad practice.
https://stackoverflow.com/a/4792296/11208809

Reply:
this is remaining